### PR TITLE
Document additional use and show an example session

### LINKED ISSUE #27: update dm_plans documentation
Working with the new documentation for dm_plans at 19ID today, found some more content..

### PR DESCRIPTION
**Pull request by prjemian**
- close #27

Example session starts here: https://github.com/BCDA-APS/bs_model_instrument/blob/27-dm_plans_docs/docs/source/guides/dm.md#example-session-1

### CODE REVIEW

File: docs/source/guides/dm.md [hunk 1]
```diff
@@ -83,3 +83,331 @@ has more support to integrate various capabilities of the DM tools.
 
 For more information about DM, see its [API
 Reference](https://git.aps.anl.gov/DM/dm-docs/-/wikis/DM/Beamline-Services/API-Reference).
+
+## Example session
+
+Let's work through an example session at an APS beamline.  The instrument
+package is named `mic_instrument`.  Start a new console session:
```
Comment (MDecarabas) [inline docs/source/guides/dm.md:90]: let's make this generalized to our package

File: docs/source/guides/dm.md
Comment (MDecarabas) [inline docs/source/guides/dm.md:1]: This seems useful, however I believe we are tending back to the monolith structure that has cause bluesky_training to be hard to use. Let's keep documentation of the package exclusively to bs_model_instrument, and instead add aspects such as these to v2 of bluesky training. I think it is important to not have scope creep happen and keep packages/repositories focused on solving 1 problem.

**Review by MDecarabas — CHANGES_REQUESTED** [review]

File: docs/source/guides/dm.md (hunk 1 above)
Comment (prjemian) [reply docs/source/guides/dm.md]: See commit e52d198c18.

**Comment by prjemian** [comment]
Ready for review again.

**Review by MDecarabas — APPROVED** [review]
